.github/workflows/tests.yml

```diff
@@ -113,8 +113,8 @@ jobs:
           name: bazel-testlogs
           path: bazel-testlogs
 
-  buildifier:
-    name: Check Starlark and Docs
+  lint:
+    name: Check Formatting and Docs
     runs-on: macos-12
```

Comment:
On a side note, I just wonder if this even has to run on the macOS

Reply:
Probably not good point, I'll update

Reply:
Cool you might just need to pull a different `buildifier` binary in, otherwise we can land this and followup ;